config/env/development.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
     }
   },
   app: {
-    title: defaultEnvConfig.app.title + ' - Development Environment'
+    title: defaultEnvConfig.app.title + ' - Development Environment',
+    domain: 'localhost:3000'
```

Comment:
@Koblaid We want to allow setting the domain from an env variable, or to be defined explicitly in the configuration file, which is what @mleanos was suggesting. So this change should be made:
```javascript
domain: process.env.DOMAIN || 'localhost'
```
And it should be added after the `port` configuration option so you can re-use it of course.

I'm ok with setting that only on `development.js` and `test.js` as you suggested.